egs2/librispeech/asr1/local/fstaddselfloops.pl

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env perl
```

Comment:
This looks a general fst operation. We may move it to https://github.com/espnet/espnet/tree/master/egs2/TEMPLATE/asr1/scripts In the future.